test/deploy/kubernetes/runtime-pods/components_data_node1.yaml

```diff
@@ -20,6 +20,7 @@ spec:
     metadata:
       labels:
         app: data-node1
+        cortx.io/machine-id: aaa120a9e051d103c164f605615c32a4
```

Comment:
Can we pull this value directly from pod metadata id? Instead of explicitly specifying it